.github/workflows/deploy.yml

```diff
@@ -25,18 +25,21 @@ jobs:
       run: yarn test
     - name: copy yarn.lock
       run: cp yarn.lock packages/publisher/
+    - name: prepare package outside monorepo
+      run: |
+        cp -r packages/publisher ${{ runner.temp }}/publisher
+        cd ${{ runner.temp }}/publisher
+        yarn install
     - name: deploy publisher
       uses: azure/functions-action@v1
       with:
         app-name: types-publisher
         slot-name: production
         publish-profile: ${{ secrets.TYPESPUBLISHERFUNCTIONSPUBLISHPROFILE }}
-        package: packages/publisher
-        scm-do-build-during-deployment: true
-        enable-oryx-build: true
+        package: ${{ runner.temp }}/publisher
     - name: upload artifacts
-      if: always()
+      if: failure()
```

Comment:
does `if: failure()` mean the old thing still runs as a backup?

Reply:
We were only doing this for debugging purposes, and you typically don't need to debug unless something goes wrong. The upload takes much longer now that the temp dir has a copy of the app in it.